app/src/features/surveys/components/CreateSamplingMethod.tsx

```diff
@@ -0,0 +1,44 @@
+import EditDialog from 'components/dialog/EditDialog';
+import { useState } from 'react';
+import MethodForm, {
+  ISurveySampleMethodData,
+  SamplingSiteMethodYupSchema,
+  SurveySampleMethodDataInitialValues
+} from './MethodForm';
+
+interface ISamplingMethodProps {
+  open: boolean;
+  onSubmit: (data: ISurveySampleMethodData) => void;
+  onClose: () => void;
+}
+
+const CreateSamplingMethod: React.FC<ISamplingMethodProps> = (props) => {
+  const [isSubmitting, setIsSubmitting] = useState(false);
+
+  const handleSubmit = (values: ISurveySampleMethodData) => {
+    setIsSubmitting(false);
```

Comment:
shouldn't this be `setIsSubmitting(true)`?

Reply:
the dialog doesn't have an API call so this all happens instantly. Removed the state